src/app.ts

```diff
@@ -203,13 +204,22 @@ export async function getApp(
 
   app.setErrorHandler(errorHandler)
 
+  const dependencies: DependencyOverrides = configOverrides
```

Comment:
wait, how is this different from https://github.com/lokalise/node-service-template/pull/671/files?

Reply:
It's the same thing. My branch was outdated, after merge I got the new changes from 671